dashboard/app/models/levels/standalone_video.rb

```diff
@@ -25,7 +25,6 @@
 
 class StandaloneVideo < Level
   serialized_attrs %w(
-    encrypted_teacher_markdown
```

Comment:
Did you learn why we were encrypting these?

Reply:
@joshlory looks like you encrypted it (https://github.com/code-dot-org/code-dot-org/commit/a221c26216350286d3c15d46be7016b0c2b3cd07). can you explain why?

Reply:
In talking with JoshL he doesn't know of the reason why this one was set up different from the others